### DISCUSSION TITLE
cannot access repository

### DISCUSSION THREAD

**Discussion by ayubowan2u** [community/community#75852] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I am newbie here so apologies if this question should be raised elsewhere.

I am trying to clone git@github.com:dmorand/nwfirmi.git so that I can build the Nextwindow touchscreen driver for Ubuntu 22.04 and I have been following the instructions here https://github.com/dmonad/nwfermi. 

However, when I issue the git clone command I get the following response

_ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists._

Would anyone be able to give me some pointers as to whether the issue is related to wrong url for the repository or whether it is an access problem?

Many thanks.

**Comment by Habibsultani** [comment]
Hi try this command: 
`git clone https://github.com/dmonad/nwfermi.git`

**Reply by ayubowan2u** [reply]
Hi, thanks for pointing out the typo by the owner of the repo. 

It was downloaded without any issues once I changed the git command as pointed out by you.

Many thanks for the prompt response.

**Comment by Habibsultani** [comment]
You are getting this error because you did not write the cloning link correctly and Github search engine can not find that repository.
